py/torch_tensorrt/dynamo/runtime/_TorchTensorRTModule.py

```diff
@@ -9,7 +12,7 @@
 logger = logging.getLogger(__name__)
 
 SerializedTensorRTEngineFmt = Tuple[
-    str, str, str, bytes, str, str, str
+    str, str, str, bytes, str, str, bytes, str
```

Comment:
The last two `bytes` and `str` correspond to hardware_compatible and serialized_metadata ?

Reply:
Nice catch. I changed the order but forget to change this definition. My apology.